### ISSUE TITLE
Nuke Ops pls fix?

### ISSUE THREAD

**Issue by penguinfisher** [yogstation13/yogstation-classic#330]
Everytime I play ops I lose to some horseshit like a borg opening the shuttle door and sending it away. A borg bound to the station should NOT have access to an enemy corporation's electronics. Or I lose because our nuke gets spaced because some assistant with a space suit and welder breaks down the bottom walls and leaves it.

Wouldn't it be nice if the shuttle walls were unbreakable from the outside? or borgs couldn't open the door? If you steal an agent card and you break in that's on the ops; not bullshit.

It seems like nuke ops can be single-handedly stopped by an assistant with a welder and a stolen EVA suit. A crew of 5 armed agents vs one assistant. this don't seem right at all no sir no.

Also I went into crit as a cabbage on the syndie station because the light is borked :^)

tl;dr: Borgs can't open shuttle + shuttle walls can't be welded down

**Comment by ShadowDeath6** [comment]
Git gud :^)

Nuke should be more well protected, add some shutters that only ops can open or some shit.

**Comment by ghost** [comment]
Removing/spacing the nuke is a crucial part of nukeops that'd be ridiculous to block off completely.

As such, making the walls unbreakable is a bad idea.

However, I have a couple-step idea that could fix this.

Firstly, disable AI control on the nukeops shuttle terminal and blast door button. It's unfair for Nanotrasen AI's to be able to cuck it.

Secondly- 

Attatch turrets to shuttle exterior.

Not hybrid turret gun turrets like AI core. Syndicate turrets. These are already in the code-they fire bullets at anything that passes by without an agent/syndicate ID. The same bullets used in lmg's-the ones that hit for 40 damage flat and stamina.

This way, it becomes sufficiently harder for crew to break into the ops shuttle unprepared-emp's can short the turrets out, and you can of course break them with sufficient firepower or find a blind spot, but it'll be harder while being targeted by 40 damage syndicate autoturrets tracking and firing at you.

This makes it still possible to steal the nuke, but require a coordinated, efficient effort to actually do so without getting killed.

If that's not enough, make the detomatrix cartridge come with an inbuilt proximity alarm mechanic-stepping foot on the shuttle triggers a prox alarm that is sent to the military pda, tipping the leader off someone is inside. Of course, intelligent ops can do this already with a proximity alarm/signaller sensor, parts available in the ops shuttle engineering foyer.

As an alternative to the above, I have two suggestions.

The first is that again, the military PDA have an inbuilt nuclear device tracker much like he janitor's can locate their cart, to locate the nuke when an assistant steals it. If it is at all possible to absolutely block entirely the exiting of something from a zlevel, do that to the nuke, to prevent spacings and require crew to defend it.

The second is set the vault nuke to the same code as the ops shuttle nuke, preset at roundstart, without requiring an admin, this way ops can still blow the station even with nuke spaced.

**Comment by penguinfisher** [comment]
I like everything adam posted pls add ty 

no more fluke ops ragequit

**Comment by Super3222** [comment]
> It seems like nuke ops can be single-handedly stopped by an assistant

That's just the enjoyment of SS13. And I've never heard of an assistant ever doing that, unless they are armed with some actual weaponry.

Adding syndicate turrents to the syndiship would be a good idea, consider that /tg/ has already done it and it's makes more sense than average crew members being able to break into a nuclear operative team's spaceship.

**Comment by Groudonmaster** [comment]
Adam's done it like twice because he likes to. As a clown. In a space suit. Armed with just the space suit. R.I.P. Nuke

**Comment by Kmc2000** [comment]
Simple fix, already coded : 

Make the gun_turret obj buyable for 20tc, it would be via a beacon

We have syndicate defense turrets that we never use, why not add that

Hell if i new how to merge I could do that

**Comment by ShadowDeath6** [comment]
Tg has turrets on there FOR FREEEEEEE

**Comment by X-TheDark** [comment]
How many turrets would their shuttle get? Also, yeah, if this gets implemented they get it free, seriously, paying 20 TC for one is ludicrous. Even War would not fund you with nearly enough to keep you safe.

**Comment by ShadowDeath6** [comment]
I think 2 turrets on each side of the main ship and 1 turret on each corner of the pod thing.

**Comment by B1gb3ast** [comment]
If you can buy turrets getting rid pf the nuke would become horrifying.

It already takes 80% of the minimal nuke timer to disarm the thing and getting lucky on RNG if the nuke pOOPS took the disk out.

Imagine walking into el solars and finding an AI controlled turret guarding it. Pls no.

**Comment by ghost** [comment]
> If you can buy turrets getting rid pf the nuke would become horrifying.
> It already takes 80% of the minimal nuke timer to disarm the thing and getting lucky on RNG if the nuke pOOPS took the disk out.
> Imagine walking into el solars and finding an AI controlled turret guarding it. Pls no.

step 1. weld into the syndie shuttle

step 2. walk off with the nuke
